projects/RabbitMQ.Client/client/impl/InlineWorkService.cs

```diff
@@ -0,0 +1,11 @@
+﻿using RabbitMQ.Client.Impl;
+
+namespace RabbitMQ.Client.client.impl
+{
+    internal sealed class InlineWorkService : ConsumerWorkService
```

Comment:
Why inherit from ConsumerWorkService?

Reply:
because there is some fancy logic in modelbase ctor to create IConsumerDispatcher. Ideally this class should not need to exist. If I am allowed to refactor modelbase to just take IConsumerDispatcher as ctor parameter and place the dissision whats the implementation somewhere else im happy to do so. I just wanted to comply with the current patterns as close as possible.